server.js

```diff
@@ -14,14 +22,40 @@ const userRoutes = require('./routes/user');
 /* ==== Internal Modules ==== */
 
 
+
 /* ==== Instanced Modules  ==== */
 const app = express();
 
+
+// Routes
+const routes = require("./routes");
+require('./config/database');
+
 /* ====  Configuration  ==== */
 const PORT = process.env.PORT || 4000;
-
 app.set("view engine", "ejs");
 
+
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7821591)
```suggestion
    res.send("Welcome");
```